### PR TITLE
Add pytest-selenium recipe

### PR DESCRIPTION
**Pull request by birdsarah**

### CODE REVIEW

File: recipes/pytest-selenium/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  name: pytest-selenium
+  version: 1.6.0
+
+source:
+  git_rev: v1.6.0
+  git_url: https://github.com/pytest-dev/pytest-selenium.git
+
+build:
+  noarch_python: True
+  number: 1
```
Comment (nicoddemus) [inline recipes/pytest-selenium/meta.yaml]: For recipes which are basically a `setup.py install` script, I suggest you to use [build/script](http://conda.pydata.org/docs/building/meta-yaml.html#script) instead and ditch the `.bat` and `.sh` files.

```yml
build:
  number: 0
  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Here's another example: https://github.com/conda-forge/pytest-mock-feedstock/blob/master/recipe/meta.yaml#L17

Also, the first packages should start with `build number` `0`.

File: recipes/pytest-selenium/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+package:
+  name: pytest-selenium
+  version: 1.6.0
+
+source:
+  git_rev: v1.6.0
+  git_url: https://github.com/pytest-dev/pytest-selenium.git
+
+build:
+  noarch_python: True
```
Comment (nicoddemus) [inline recipes/pytest-selenium/meta.yaml]: Not sure if you should use `noarch_python` on `conda-forge` recipes, even for pure Python packages...

**Comment by birdsarah** [comment]
@nicoddemus - thanks for the comments - I've added you as a collaborator to the repo this PR is from - feel free to make changes I'm not precious about this, otherwise, will get to it later.

**Comment by ocefpaf** [comment]
It looks like you will need to package more dependencies first. This is from the CIs' log:

```
conda.exceptions.NoPackagesFoundError:
  - pytest-html >=1.7
  - pytest-variables
  - pytest-base-url
```

**Comment by nicoddemus** [comment]
@birdsarah OK thanks, will apply the changes when I have some time.

As @ocefpaf mentioned, all dependencies must already be published in conda-forge first though. Do you by any chance have already the other recipes laying around if that's the case?

Also, it is OK to add more than one recipe in a single PR, as long as they are independent from each other (@ocefpaf please correct me if I'm wrong).

**Comment by ocefpaf** [comment]
> Also, it is OK to add more than one recipe in a single PR, as long as they are independent from each other (@ocefpaf please correct me if I'm wrong).

If they are dependent is kind of OK too :smile: 

The thing is... They will be built in the proper order and dependencies downstream will have the previous package available in staged-recipes. However, when it is merged, all the recipes are built at the same time, so those that have pending dependencies will fail.

TL;DR avoid doing that unless you need them together to iterate some testing in staged-recipes. Separete PRs are usually better.

**Comment by nicoddemus** [comment]
OK thanks for the explanation!

**Comment by birdsarah** [comment]
i do have the other recipes. will start with those!

**Comment by nicoddemus** [comment]
> i do have the other recipes. will start with those!

Sounds good, feel free to cc me on them. 😁

**Comment by birdsarah** [comment]
@davehunt @nicoddemus this should pass now all dependencies are in conda-forge.

Are you both still good with being listed as maintainers on this recipe.

**Comment by nicoddemus** [comment]
Awesome, thanks!

**Comment by birdsarah** [comment]
ping @msarahan - what do you think about merging this?
